isPrime.js

```diff
@@ -0,0 +1,29 @@
+// A function that checks if a given number is prime or not
+
+const isPrime = (number) => {
+  // Check if number is a real number
+  if (typeof number !== "number") return false;
+
+  // Check factor 1: The smallest number that satisfies the definition of prime numbers is 2
```

Comment:
A comment is not needed on this line. also as it has only a return statement, you can put it on the same line